packages/components/src/Chip/Chip.css

```diff
@@ -0,0 +1,32 @@
+.chip {
+  --chip-border-size: var(--space-minuscule);
+  --chip-border-radius: var(--space-small);
+  --chip-border-color: var(--color-blue--light);
+  --chip-text-color: var(--color-blue);
+  --chip-item-spacing: var(--space-small);
+  --chip-padding--vertical: var(--space-smallest);
+  --chip-padding--horizontal: var(--space-small);
+  --chip-padding: var(--chip-padding--vertical) var(--chip-padding--horizontal);
+  --chip-shadow: var(--shadow-low);
+  --chip-shadow-hover: var(--shadow-base);
+
+  display: flex;
+  max-width: fit-content;
+  box-shadow: var(--chip-shadow);
+  padding: var(--chip-padding);
+  border: var(--chip-border-size) solid var(--chip-border-color);
+  border-radius: var(--chip-border-radius);
+  align-items: center;
+  gap: var(--chip-item-spacing);
+}
+
+.chip:hover {
+  box-shadow: var(--chip-shadow-hover);
+  transition: all var(--timing-base) ease-in-out;
+}
```

Comment:
Kinda odd for it to have a hover effect but not be clickable. @chris-at-jobber was the chip itself meant to be clickable?

Reply:
Chip should be clickable and focusable - it's a pretty versatile component. @joelwarrington-jobber can you look at the Chip in the Jobber codebase as a reference?

Reply:
The Figma file I was referencing didn't have a lot of the considerations written out which has led to me assuming functionality that is not directly visible. Have started investigations into our internal Chip component and see what changes can be made from there